src/images/extract.py

```diff
@@ -216,7 +216,7 @@ def replace(tag):
         elif img_src.startswith("//bits.wikimedia.org/"):
             dsk_url = img_src[46:]
 
-        elif img_src.startswith("//upload.wikimedia.org/wikipedia/es/"):
+        elif img_src.startswith("//upload.wikimedia.org/wikipedia/%s/" % config.LANGUAGE):
             dsk_url = img_src[36:]
```

Comment:
The length of the `config.LANGUAGE` param may vary (examples: `es`, `azb`, `simple`, `be-tarask`, `zh-min-nan`).
This should be taken into account when removing the prefix.